data/help/texts.json

```diff
@@ -247,12 +247,12 @@
     "messages": [
       "~       Liquid\n%%       Food\n!       Medication\nThese are all consumed by using <press_eat>.  They provide a certain amount of nutrition, quench your thirst, may be a stimulant or a depressant, and may provide morale.  There may also be more subtle effects.",
       "/       Large weapon\n;       Small weapon or tool\n,       Tiny item\nThese are all generic items, useful only to be wielded as a weapon.  However, some have special uses; they will show up under the TOOLS section in your inventory.  Press <press_apply> to use these.",
-      ")       Container\nThese items may hold other items.  Some are passable weapons.  Many will be listed with their contents, e.g. \"plastic bottle of water\".  Those containing comestibles may be eaten with <press_eat>; this may leave an empty container.",
+      ")       Container\nThese items may hold other items.  Some are passable weapons.  Many will be listed with their contents, e.g. \"plastic bottle > water\".  Those containing comestibles may be eaten with <press_eat>; this may leave an empty container.",
       "[       Clothing\nThis may be worn with the <press_wear> key or removed with the <press_take_off> key.  It may cover one or more body parts; you can wear multiple articles of clothing on any given body part, but this will encumber you severely.  Each article of clothing may provide storage space, warmth, encumbrance, and a resistance to bashing and/or cutting attacks.  Some may protect against environmental effects.",
```

Comment:
```suggestion
      "[       Clothing\nThis may be worn with the <press_wear> key or removed with the <press_take_off> key.  It may cover one or more body parts; you can wear multiple articles of clothing on any given body part, but this will encumber you severely.  Each article of clothing may provide storage space, warmth, encumbrance, and resistance to bashing and/or cutting attacks.  Some may protect against environmental effects.",
```